src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -518,14 +518,20 @@ export const exchangeStitchingEnvironment = ({
             __typename
           }`,
           resolve: async (_source, args, context, info) => {
-            return await info.mergeInfo.delegateToSchema({
+            const orderResult = await info.mergeInfo.delegateToSchema({
               schema: exchangeSchema,
               operation: "query",
               fieldName: "commerceMyOrders",
```

Comment:
**thought:** Alternatively, we could make this change in [Exchange's `myOrders` type](https://github.com/artsy/exchange/blob/c4d6bb4af8da22de05aa15eb4f9724194e10145c/app/graphql/types/query_type.rb#L70-L78).